sites/geohub/src/routes/(map)/dashboards/electricity/components/Charts.svelte

```diff
@@ -1,10 +1,12 @@
 <script lang="ts">
+	import { LineChart } from '@carbon/charts-svelte';
+	import '@carbon/charts-svelte/styles.css';
```

Comment:
Is @carbon/charts-svelte necessary? GeoHub is not using IBM's carbon design system, it is better if you can just use pure d3 to implement a chart diagram.
Mixing with a different design system might have conflicts between Bulma and UNDP design system.